ly2video/cli.py

```diff
@@ -1103,19 +1129,17 @@ def findExecutableDependencies(options):
 
     redirectToNull = " >%s" % portableDevNull()
 
-    ffmpeg = options.winFfmpeg + "ffmpeg"
-    if os.system(ffmpeg + " -version" + redirectToNull) != 0:
-        fatal("FFmpeg was not found (maybe use --windows-ffmpeg?).", 2)
+    stdout = safeRun(["ffmpeg", "-version", redirectToNull],
```

Comment:
Similarly here, why not just have `--ffmpeg` and `--timidity` options which allow pointing to those executables in any location on any OS?